schemas/telemetry/crash/crash.4.schema.json

```diff
@@ -1272,6 +1272,10 @@
               "description": "<size>, Windows-only, available physical memory in bytes",
               "type": "string"
             },
+            "AvailableSwapMemory": {
+              "description": "<size>, Amount of free swap space in bytes. - Under macOS, populated with the contents of\n  sysctl \"vm.swapusage\" :: xsu_avail.\n- Under Linux, populated with /proc/meminfo's SwapFree. - Not available on other platforms.",
```

Comment:
I modified these from the output to correspond with other fields of similar kind in the file.